dev-tools/ecs-migration.yml

```diff
@@ -400,39 +334,36 @@
 - from: nginx.access.http_version
   to: http.version
   alias: true
-  copy_to: false
 
 - from: nginx.access.geoip.continent_name
   to: source.geo.continent_name
   alias: true
-  copy_to: false
 
 - from: nginx.access.geoip.country_iso_code
   to: source.geo.country_iso_code
   alias: true
-  copy_to: false
 
 - from: nginx.access.geoip.location
   to: source.geo.location
   alias: true
-  copy_to: false
 
 - from: nginx.access.geoip.region_name
   to: source.geo.region_name
   alias: true
-  copy_to: false
 
 - from: nginx.access.geoip.city_name
   to: source.geo.city_name
   alias: true
-  copy_to: false
 
 - from: nginx.access.geoip.region_iso_code
   to: source.geo.region_iso_code
   alias: true
-  copy_to: false
 
 - from: nginx.access.agent
   to: user_agent.original
   alias: true
-  copy_to: false
+
+# From Auditbeat's auditd module.
```

Comment:
@andrewkroh can you double check if that is correct?

Reply:
Why introduce this in this PR? I think a full review of Auditbeat GA modules is required, and the field mapping will be added at that time.

Reply:
Gotcha